src/xdocs/config_javadoc.xml

```diff
@@ -233,6 +233,122 @@ public class TestClass {
       </subsection>
     </section>
 
+    <section name="JavadocBlockTagLocation">
+      <p>Since Checkstyle 8.24</p>
+      <subsection name="Description" id="JavadocBlockTagLocation_Description">
+        <p>
+          Checks that a
+          <a href="https://docs.oracle.com/en/java/javase/11/docs/specs/doc-comment-spec.html#block-tags">
+          javadoc block tag</a> appears only at the beginning of a line, ignoring leading asterisks,
+          white space, and the initial comment delimiter (&#47;**).
+        </p>
+        <p>
+          Rationale: all Javadoc block tags not placed at the beginning of a line
+          are treated as the plain text. This may be intentional, but changing
+          the text may unexpectedly place them at the beginning of the line and
+          they will no longer be recognized as plain text.
+          Conversely, if a block tag is accidentally moved to the middle of a line,
+          it will be treated as plain text without any violations, but the generated
+          documentation will be erroneous.
+        </p>
+      </subsection>
+
+      <subsection name="Properties" id="JavadocBlockTagLocation_Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>tags</td>
+            <td>Specify the javadoc tags to process.</td>
+            <td><a href="property_types.html#stringSet">String Set</a></td>
+            <td><code>exception, hidden, author, deprecated, throws, version, see,
+              serialData, param, serial, provides, serialField, uses, return, since
+            </code></td>
+            <td>8.24</td>
+          </tr>
+          <tr>
+            <td>violateExecutionOnNonTightHtml</td>
+            <td>
+              Control when to print violations if the Javadoc being examined by this check
+              violates the tight html rules defined at
+              <a href="writingjavadocchecks.html#Tight-HTML_rules">Tight-HTML Rules</a>.</td>
+            <td><a href="property_types.html#boolean">Boolean</a></td>
+            <td><code>false</code></td>
+            <td>8.24</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples" id="JavadocBlockTagLocation_Examples">
```

Comment:
Please add code examples and somewhere specify how to fix false positives.

Example:
````
-     * '@serial' literal in @serial Javadoc tag.
+     * '@serial' literal in {@code @serial} Javadoc tag.
````

Reply:
done

Reply:
Specific examples were not added to Examples section.
Please add code showing example of cases that would produce a violation.
Example: https://checkstyle.sourceforge.io/config_naming.html#AbbreviationAsWordInName_Examples

Reply:
done